[2025-10-23T21:24:15.813+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_branch_dop_operator_v3.should_run scheduled__2025-10-23T21:23:00+00:00 [queued]>
[2025-10-23T21:24:15.817+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_branch_dop_operator_v3.should_run scheduled__2025-10-23T21:23:00+00:00 [queued]>
[2025-10-23T21:24:15.817+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-10-23T21:24:15.823+0000] {taskinstance.py:1382} INFO - Executing <Task(_BranchPythonDecoratedOperator): should_run> on 2025-10-23 21:23:00+00:00
[2025-10-23T21:24:15.827+0000] {standard_task_runner.py:57} INFO - Started process 141 to run task
[2025-10-23T21:24:15.831+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'example_branch_dop_operator_v3', 'should_run', 'scheduled__2025-10-23T21:23:00+00:00', '--job-id', '1061', '--raw', '--subdir', '/home/airflow/.local/lib/python3.11/site-packages/airflow/example_dags/example_branch_python_dop_operator_3.py', '--cfg-path', '/tmp/tmpof4nnzkc']
[2025-10-23T21:24:15.832+0000] {standard_task_runner.py:85} INFO - Job 1061: Subtask should_run
[2025-10-23T21:24:15.882+0000] {task_command.py:416} INFO - Running <TaskInstance: example_branch_dop_operator_v3.should_run scheduled__2025-10-23T21:23:00+00:00 [running]> on host 39b6b9dcf23a
[2025-10-23T21:24:15.924+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='example_branch_dop_operator_v3' AIRFLOW_CTX_TASK_ID='should_run' AIRFLOW_CTX_EXECUTION_DATE='2025-10-23T21:23:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-23T21:23:00+00:00'
[2025-10-23T21:24:15.928+0000] {logging_mixin.py:154} WARNING - /home/airflow/.local/lib/python3.11/site-packages/airflow/utils/context.py:314 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-10-23T21:24:15.928+0000] {logging_mixin.py:154} INFO - ------------- exec dttm = 2025-10-23T21:23:00+00:00 and minute = 23
[2025-10-23T21:24:15.928+0000] {python.py:194} INFO - Done. Returned value was: empty_task_2
[2025-10-23T21:24:15.928+0000] {python.py:227} INFO - Branch callable return empty_task_2
[2025-10-23T21:24:15.928+0000] {skipmixin.py:173} INFO - Following branch empty_task_2
[2025-10-23T21:24:15.937+0000] {skipmixin.py:239} INFO - Skipping tasks [('empty_task_1', -1)]
[2025-10-23T21:24:15.958+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=example_branch_dop_operator_v3, task_id=should_run, execution_date=20251023T212300, start_date=20251023T212415, end_date=20251023T212415
[2025-10-23T21:24:15.968+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-10-23T21:24:15.982+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
